web/pdf_page_view.js

```diff
@@ -260,6 +260,10 @@ class PDFPageView {
    */
   updatePosition() {
     if (this.textLayer) {
+      let readableStream = this.pdfPage.streamTextContent({
+        normalizeWhitespace: true,
+      });
+      this.textLayer.setTextContentStream(readableStream);
```

Comment:
okay, plan is to change if above to `if (this.textLayer && !this.textLayer.renderingDone) {`